boards/xtensa/esp32/esp32-wrover-kit/src/esp32_bringup.c

```diff
@@ -259,6 +263,20 @@ int esp32_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_SENSORS_BMP180
+  /* Try to register BMP180 device in I2C0 */
+
+  ret = board_bmp180_initialize(0, 0);
+
+  if (ret < 0)
+    {
+      syslog(LOG_ERR, "Failed to initialize BMP180"
+                       "Driver for I2C0: %d\n", ret);
+      ret = board_bmp180_initialize(0, 1);
```

Comment:
The issue raised by @Ouss4 has been marked as resolved, but the code still tries to initialize the sensor on the other I2C bus as a fallback.
This mapping should be specified via Kconfig at the board configuration section.

Reply:
@gustavonihei , sorry fot that. I've just fixed it in the code. Please, review it again.